core/src/com/unciv/logic/city/CityFocus.kt

```diff
@@ -24,28 +26,31 @@ enum class CityFocus(
     binding: KeyboardBinding? = null
 ) : IsPartOfGameInfoSerialization {
     // region Enum values
-    NoFocus("Default Focus", true, null) {
+    NoFocus("Default", true, null) {
         override fun getStatMultiplier(stat: Stat) = 1f  // actually redundant, but that's two steps to see
     },
-    FoodFocus("[${Stat.Food.name}] Focus", true, Stat.Food),
-    ProductionFocus("[${Stat.Production.name}] Focus", true, Stat.Production),
-    GoldFocus("[${Stat.Gold.name}] Focus", true, Stat.Gold),
-    ScienceFocus("[${Stat.Science.name}] Focus", true, Stat.Science),
-    CultureFocus("[${Stat.Culture.name}] Focus", true, Stat.Culture),
-    GoldGrowthFocus("Gold Growth Focus", false) {
+    Manual("Manual", true, null) {
+        override fun getStatMultiplier(stat: Stat) = 1f
+    },
+    FoodFocus("[${Stat.Food.character}]", true, Stat.Food),
```

Comment:
Why are all these wrapped in square brackets? There should be no translation going on here anyway

Reply:
Good point